Asyncio_rmq/Asyncio_rmq/asyncio_rmq.py

```diff
@@ -0,0 +1,350 @@
+"""
+Asyncio with RabbitMQ.
+
+Parameters
+----------
+role (-role or -r): string
+    Role for the instance to play.
+    Option 1: "start"
+    Option 2: "listen"
+
+exchange (-exchange or -e): string
+    Name for the RabbitMQ exchange. Can be left as default.
+
+delay(-delay or -d): integer
+    Delay (in seconds) that can be applied between responses. This is only to make the interactions
+    between the two nodes easier to follow. Default is 3 seconds.
+
+Return: None
+"""
+import argparse
+import asyncio
+import aio_pika
+
+
+async def createConnection(Queue, routing_key, loop):
+    """
+    Create RabbitMQ Connection.
+
+    Parameters
+    ----------
+    Queue: List[string]
+        Names of queues to create for message exchange.
+
+    routing_key: List[string]
+        Names of routing keys to use. In this example it mimics the Queue names.
+
+    loop: Asyncio object
+        Asyncio event loop
+
```

Comment:
My suggestion is to put the type hints in the argument list where `mypy` and your IDE can parse them. Then the descriptions look a bit cleaner as well. Like so:

```suggestion
async def createConnection(Queue: List[string], routing_key: List[string], loop: asyncio.AbstractEventLoop):
    """
    Create RabbitMQ Connection.

    Parameters
    ----------
    Queue
        Names of queues to create for message exchange.
    routing_key
        Names of routing keys to use. In this example it mimics the Queue names.
    loop
        Asyncio event loop
```